### DISCUSSION TITLE
Quarkus 3.0.0.Final core artifacts are available

### DISCUSSION THREAD

**Discussion by gsmet** [quarkiverse/quarkiverse#108] [General]
Hi Quarkiverse contributors,

The Quarkus 3.0.0.Final core artifacts are now available on Maven Central and you should be able to upgrade your Quarkiverse extensions to Quarkus 3.0.0.Final and release a first stable version targeting Quarkus 3.0.0.Final.

Ideally, this release should be done before April 25th so your extensions are available when we release the full Quarkus Platform and announce Quarkus 3.0 on April 26th.

If you encounter any issue while doing so, please ping @gsmet and @gastaldi , we will be happy to help. If you want us to handle the upgrade and the release, that's possible too, just ping us.

Thanks for your cooperation, let's make the Quarkus 3 release a great success together!

-- 
Guillaume

**Comment by alexeysharandin** [comment]
All my extentions updated. Ready for release :)
